test/NuGet.Core.Tests/NuGet.Versioning.Test/VersionRangeTests.cs

```diff
@@ -287,6 +287,22 @@ public void VersionRange_Exact()
             Assert.True(versionInfo.Satisfies(NuGetVersion.Parse("4.3.0")));
         }
 
+        [Theory]
+        [InlineData("1", "1.0")]
+        [InlineData("1", "1.0.0.0")]
+        [InlineData("123.456", "123.456.0.0")]
+        [InlineData("[2021,)", "[2021.0.0.0,)")]
+        [InlineData("[,2021)", "[,2021.0.0.0)")]
+        public void VersionRange_MissingVersionComponents_AssumedToBeZero(string shortVersionSpec, string longVersionSpec)
```

Comment:
Just in case can you add some edge case unit tests like `-1`, `0`, `,` ,`-`, `+`,`02`, `1.`,` .1`, `1,`,`,1`  etc for happy or unhappy path since they were excluded previously?

Reply:
I assumed edge cases were already being tested, so I basically focused on the fact that shorter (but valid) version specs were expanded with as many `.0` components as needed.

Anyway I added a few test cases. Let me know if you'd like to see more.

Reply:
Thank you for your contribution and addressing the PR commend.
I don't think there're many unit tests shorter than 3 chars. Just in case add below few then I believe we're good.
`,` ,`-`, `+`, `a`

Reply:
Done